packages/cronie/fix-default-environment.patch

```diff
@@ -21,18 +21,18 @@ diff -uNr cronie-1.5.5/src/env.c cronie-1.5.5.mod/src/env.c
 -		"LANGUAGE",
  		"RANDOM_DELAY",
 +		"ANDROID_ART_ROOT",
-+		"ANDROID_DATA",
-+		"ANDROID_I18N_ROOT",
-+		"ANDROID_ROOT",
-+		"ANDROID_RUNTIME_ROOT",
-+		"ANDROID_TZDATA_ROOT",
-+		"BOOTCLASSPATH",
-+		"DEX2OATBOOTCLASSPATH",
-+		"EXTERNAL_STORAGE",
-+		"LD_PRELOAD",
-+		"PREFIX",
-+		"TMPDIR",
-+		"TZ",
++               "ANDROID_DATA",
++               "ANDROID_I18N_ROOT",
++               "ANDROID_ROOT",
++               "ANDROID_RUNTIME_ROOT",
++               "ANDROID_TZDATA_ROOT",
++               "BOOTCLASSPATH",
++               "DEX2OATBOOTCLASSPATH",
++               "EXTERNAL_STORAGE",
++               "LD_PRELOAD",
++               "PREFIX",
++               "TMPDIR",
++               "TZ",
```

Comment:
Use tabs not spaces

Reply:
Your editor probably supports `.editorconfig`.
https://editorconfig.org/

We have one in the repository to automatically apply the right settings.